source/fundamentals/connection/connect.txt

```diff
@@ -50,13 +51,15 @@ connect to a MongoDB deployment. It instructs the driver on how it should
 connect to MongoDB and how it should behave while connected. The following
 figure explains each part of a sample connection URI:
 
-.. figure:: /includes/figures/connection_uri_parts.png
-   :alt: Connection String parts figure
+.. figure:: /includes/figures/dns_seedlist_connection_string_parts.png
+   :alt: Each part of the connection string
```

Comment:
We should use better alt text here for accessibility. It should be meaningful to someone who can't see this image. Something like:
```suggestion
   :alt: An example of a connection string that demonstrates the protocol, credentials, hostname or IP, port, and connection options
```